CONTRIBUTING.md

```diff
@@ -0,0 +1,115 @@
+# Contributuing
+
+Contributions to this project are welcome, whether minor updates to documentation or entirely new features.
+This document outlines the technical and procedural requirements for your work to be included.
+All contributions, in any form, must also confirm with the [code of conduct](CODE_OF_CONDUCT.md).
```

Comment:
JOKE: TODO: Code of Resistance. you know, to counteract the conductivity.

Reply:
heh - did get a chuckle out of that one!